aws/resource_aws_security_group_test.go

```diff
@@ -1080,11 +1119,14 @@ func TestAccAWSSecurityGroup_DefaultEgress_VPC(t *testing.T) {
 }
 
 func TestAccAWSSecurityGroup_DefaultEgress_Classic(t *testing.T) {
-
-	// Classic
 	var group ec2.SecurityGroup
+
+	oldvar := os.Getenv("AWS_DEFAULT_REGION")
+	os.Setenv("AWS_DEFAULT_REGION", "us-east-1")
+	defer os.Setenv("AWS_DEFAULT_REGION", oldvar)
```

Comment:
IMO this is the wrong approach; can we set a global test variable which is picked up, rather than an Env Variable which could break other tests when ran in parallel in the future?

Reply:
I'm not sure the original rationale for this, but this is how its done across the rest of the provider. 🙁  If we should change the implementation, we should create a separate issue as it will probably require some acceptance testing framework rework.